pallets/interest-accrual/src/lib.rs

```diff
@@ -433,8 +422,7 @@ pub mod pallet {
 		pub(crate) fn validate_rate(interest_rate_per_year: T::InterestRate) -> DispatchResult {
 			let four_decimals = T::InterestRate::saturating_from_integer(10000);
 			ensure!(
-				interest_rate_per_year < One::one()
-					&& interest_rate_per_year > Zero::zero()
+				interest_rate_per_year > Zero::zero()
```

Comment:
I've needed to remove the upper limit because add rates overpass this limitation. In fact, I do not understand why this upper limit

Reply:
> Why this upper limitation?

Not sure exactly what should be a good upper limitation, but we need to set one to avoid the user entering here a high value that causes later an overflow in the `pow`.